flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -417,11 +424,18 @@ public Task(
                         .toArray(new IndexedInputGate[0]);
 
         this.inputGates = new IndexedInputGate[gates.length];
+        this.throughputMeter =
+                new ThroughputMeter(
+                        SystemClock.getInstance(),
+                        new EMAThroughputCalculator(
+                                taskConfiguration.get(THROUGHPUT_EMA_LAST_VALUES)));
```

Comment:
One more sign of the problem with the configuration. I don't sure that we should get this directly from taskConfiguration but I don't now another way for now.